### ISSUE TITLE
export/import from an external kong api gateway

### ISSUE THREAD

**Issue by senthil-nathan** [mybuilder/kongfig#94]
Is it possible to dump kong configuration using kongfig when its deployed externally (not localhost) ?

**Comment by kevinpgrant** [comment]
@senthil-nathan yep, this works, I use it all the time to talk to non-localhosts - try `kongfig dump --https --host api.example.com:8001 --format screen` - should work as long as the machine with kongfig on can speak to the machine with kong (firewall ports open etc) - see `kongfig help dump` for all the options, you may need to drop the --https flag or edit the port number as appropriate for your installation
